.github/workflows/test.yml

```diff
@@ -0,0 +1,95 @@
+name: Test
+
+on: [push, pull_request]
+
+jobs:
+  js-bootstrap:
+    name: Test JS bootstrap on Node v${{ matrix.node-version }}
+    runs-on: ubuntu-18.04
+
+    strategy:
+      matrix:
+        node-version: [ 8.x, 10.x, 12.x ]
```

Comment:
Does it work on 14 and 16? If so we should test those too.

Reply:
Switched matrix to test with 12, 14, 16.